src/app/store/slices/user.ts

```diff
@@ -4,13 +4,15 @@ import { authApi } from "@api";
 
 interface UserState {
   isAuth: boolean;
+  role: string;
   first_name: string | null;
   last_name: string | null;
   token: string | null;
 }
 
 const initialState: UserState = {
   isAuth: false,
+  role: "buyer",
```

Comment:
Put for default user settings correct role from union type.

Reply:
complete